### PR TITLE
Exercise and Diet

### PR DESCRIPTION
**Pull request by MikeD317**
Please review this, @casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2394)

### CODE REVIEW

File: english/exercise-and-diet/exercise-and-diet.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Actually, you can out-exercise a bad diet.
```
Comment (casey-collab) [inline english/exercise-and-diet/exercise-and-diet.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by MikeD317** [comment]
Please review this, @casey-collab

File: english/exercise/exercise-and-diet/exercise-and-diet.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Actually, you can't out-exercise a bad diet.
```
Comment (casey-collab) [inline english/exercise/exercise-and-diet/exercise-and-diet.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: english/exercise-and-diet/exercise-and-diet.md (hunk 1 above)
Comment (casey-collab) [inline english/exercise-and-diet/exercise-and-diet.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by MikeD317** [comment]
Please review this, @casey-collab

File: english/exercise/exercise-and-diet/exercise-and-diet.md (hunk 2 above)
Comment (casey-collab) [inline english/exercise/exercise-and-diet/exercise-and-diet.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: english/exercise-and-diet/exercise-and-diet.md (hunk 1 above)
Comment (casey-collab) [inline english/exercise-and-diet/exercise-and-diet.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by MikeD317** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :icecream:

**Pull request merged**